x-pack/legacy/plugins/uptime/common/constants/constants.ts

```diff
@@ -5,3 +5,5 @@
  */
 
 export const UNNAMED_LOCATION = 'Unnamed-location';
+
+export const UPTIME_INDEX_PATTERN = 'heartbeat-8*';
```

Comment:
Not sure if you realized we already have https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/uptime/common/constants/index_names.ts which has this value.

Reply:
my bad.